src/app/containers/Brand/index.test.jsx

```diff
@@ -42,5 +77,42 @@ describe(`BrandContainer`, () => {
 
       expect(skipLinkHref).toBe('#content');
     });
+
+    describe('preffered variant cookie', () => {
+      let container;
+
+      beforeEach(() => {
+        container = render(
+          BrandContainerWithContext({
+            ...variantServiceContextStub,
+          }),
+        ).container;
```

Comment:
Could we also set `scriptLink` here
```suggestion
scriptLink = container.querySelector('a[data-variant="test"]');
```